### PR TITLE
Remove cargo-casper from casper-node repository.

### LINKED ISSUE #2286: move cargo-casper own repo

### PR DESCRIPTION
**Pull request by nilz3ro**
* Removes `cargo-casper`.
* closes #2286.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,6 @@ The Casper MainNet is live.
 
 ## Get Started with Smart Contracts
 - [Writing Smart Contracts](https://docs.casperlabs.io/en/latest/dapp-dev-guide/index.html)
-- [Rust Smart Contract SDK](https://crates.io/crates/cargo-casper)
```
Comment (Fraser999) [inline README.md]: I think it's OK to leave this link here - it should remain valid.

**Review by Fraser999 — APPROVED** [review]
Approved, assuming the single nitpick is addressed.

File: README.md (hunk 1 above)
Comment (nilz3ro) [reply README.md]: fixed.

**Review by Fraser999 — APPROVED** [review]

**Pull request merged**